DEVELOPER_GUIDE.md

```diff
@@ -79,4 +79,12 @@ Sometimes you need to launch a cluster with more than one OpenSearch server proc
 You can do this by running `./gradlew run -PnumNodes=<numberOfNodesYouWant>`
 
 You can also debug a multi-node cluster, by using a combination of above multi-node and debug steps.
-But, you must set up debugger configurations to listen on each port starting from `5005` and increasing by 1 for each node.
\ No newline at end of file
+But, you must set up debugger configurations to listen on each port starting from `5005` and increasing by 1 for each node.
+
+### Backports
+
+The Github workflow in [`backport.yml`](.github/workflows/backport.yml) creates backport PRs automatically when the
+original PR with an appropriate label `backport <backport-branch-name>` is merged to main with the backport workflow
+run successfully on the PR. For example, if a PR on main needs to be backported to `1.x` branch, add a label
```

Comment:
Is it possible to add multiple labels to backport to multiple branches, like `1.x`, `1.2` ?

Reply:
Yes multiple backports also work. Adding multiple tags will create multiple PRs for each branch.